cve_bin_tool/extractor.py

```diff
@@ -56,23 +47,21 @@ def can_extract(self, filename):
                 return True
         return False
 
-    @classmethod
-    async def extract_file_tar(cls, filename, extraction_path):
+    @staticmethod
+    async def extract_file_tar(filename, extraction_path):
         """ Extract tar files """
-        try:
+        with ErrorHandler(mode=ErrorMode.Ignore) as e:
             await aio_unpack_archive(filename, extraction_path)
-            return 0
-        except Exception:
-            return 1
+        return e.exit_code
```

Comment:
This is static method because we don't need class attribute here. We are just ignoring error